doc/manual/expressions/builtins.xml

```diff
@@ -504,6 +513,28 @@ stdenv.mkDerivation { … }
         <programlisting>builtins.fetchGit {
   url = "ssh://git@github.com/nixos/nix.git";
   ref = "master";
+}</programlisting>
+      </example>
+
+      <example>
+        <title>Fetching a given revision from a repo and validate its sha256</title>
+        <para>
+          <function>builtins.fetchGit</function> can create a fixed-output
```

Comment:
Expressions that are fetched with `builtins.fetchGit` and `builtins.fetchTarball` are not derivations if I understand correctly.

Reply:
Ah you're right. As far as I understand this, those are actually paths, so "fixed-output path" (the needed function from the store's API is named `makeFixedOutputPath` btw) might be a better choice here?

Reply:
That phrase should be avoided (since they're not outputs), "content-addressed" is more universal.

Reply:
Agreed. Avoiding the term "fixed-output" in this context might prevent even more people from confusing that just as I did :)

Reply:
Using the term `content-addressable path` now.